tests/compile.sh

```diff
@@ -58,8 +58,8 @@ if [[ $MACHINE_ID == macosx ]] || [[ $MACHINE_ID == linux ]]; then
   source $PATHTR/modulefiles/ufs_${MACHINE_ID}.${RT_COMPILER}
 else
   # Activate lua environment for gaea c5
-  if [[ $MACHINE_ID == gaea-c5 ]]; then
-    source /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.sh
+  if [[ $MACHINE_ID == gaea ]]; then
+    source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh
```

Comment:
@natalie-perlin is the `source /gpfs/f5/epic/scratch/role.epic/contrib/Lmod_init_C5.sh` still needed here? i know you mentioned that we do not need to source this file in `module-setup.sh` anymore, and i've replaced it there with `module reset`. thanks!

Reply:
@ulmononian I've changed mine to module reset to match yours.